priv/repo/seeds.exs

```diff
@@ -133,3 +133,15 @@ CoursePlanner.Repo.delete_all CoursePlanner.User
       required: false
     })
 |> CoursePlanner.Repo.insert!
+
+%CoursePlanner.SystemVariable{}
+|> CoursePlanner.SystemVariable.changeset(
+    %{
+      key: "DISABLE_NOTIFICATION",
+      value: "true",
+      type: "boolean",
+      editable: true,
+      visible: true,
+      required: false
```

Comment:
however I am not sure if ecto allows that for boolean but i assume it would.
check the system_variable.ex for the detail function of :required field